src/Service.Tests/OpenApiDocumentor/CLRtoJsonValueTypeUnitTests.cs

```diff
@@ -99,12 +99,12 @@ private static IEnumerable<object[]> GetTestData_SupportedSystemTypesMapToJsonVa
     [DataRow(typeof(bool?))]
     [DataRow(typeof(char?))]
     [DataRow(typeof(Guid?))]
+    [DataRow(typeof(TimeOnly?))]
     [DataRow(typeof(TimeSpan?))]
-    [DataRow(typeof(DateTime?))]
-    [DataRow(typeof(DateTimeOffset?))]
     [DataTestMethod]
     public void ResolveUnderlyingTypeForNullableValueType(Type nullableType)
     {
         Assert.AreNotEqual(notExpected: JsonDataType.Undefined, actual: TypeHelper.GetJsonDataTypeFromSystemType(nullableType));
+        Assert.IsNotNull(TypeHelper.GetDbTypeFromSystemType(nullableType));
```

Comment:
why is this assert needed? `TypeHelper.GetDbTypeFromSystemType` never returns null.